my_list.rb

```diff
@@ -0,0 +1,27 @@
+require_relative 'my_enumerable'
+
+class MyList
+  attr_reader :list
+
+  include MyEnumerable
+
+  def initialize(*list)
+    @list = list
+  end
+
+  def each(&block)
+    @list.each(&block)
+  end
```

Comment:
- [ ] It will be great to ensure that you create your own` each` method. What I mean is that you should not use ruby defined each method but rather create your own. This is major requirement as pointed out by the previous reviewer. ⬇️ 
![Screenshot 2023-11-03 at 09 59 37](https://github.com/Megagig/Ruby-Enumerable/assets/32361963/72e9e992-44ad-45f0-b318-3ceffee27565)

Reply:
@peterrobert  Thank you so much for pointing this out. this has been corrected